lib/metrix.ex

```diff
@@ -22,6 +24,18 @@ defmodule Metrix do
     end
   end
 
+  defp initial_modifiers do
+    %{}
+    |> metric_prefix_modifier
+  end
+
+  defp metric_prefix_modifier(modifiers) do
+    case Application.get_env(:metrix, :metric_prefix) do
```

Comment:
Thinking that having the `:metric_prefix` (or, now, `:prefix`) atom in both modules is a bit of a code smell.